ksonnet-gen/kubeversion/data.go

```diff
@@ -68,6 +68,71 @@ var versions = map[string]versionData{
 			"APIVersions":                      "apiVersions",
 			"ServerAddressByClientCIDR":        "serverAddressByClientCidr",
 		},
+		constructorSpecs: map[string][]CustomConstructorSpec{
+			"io.k8s.kubernetes.pkg.api.v1.Container": []CustomConstructorSpec{
+				newConstructor("new", newParam("name"), newParam("image")),
```

Comment:
I am definitely not opposed to this, but before I make the change, does the fact that we check that the property exists before we emit the Jsonnet change your thinking on this issue?